src/components/item/sharing/CreateItemMembershipForm.tsx

```diff
@@ -47,12 +46,13 @@ const CreateItemMembershipForm = ({ item, members }: Props): JSX.Element => {
   >(MUTATION_KEYS.SHARE_ITEM);
   const { t: translateBuilder } = useBuilderTranslation();
 
-  // todo: use an array to later allow sending multiple invitations
-  const [invitation, setInvitation] = useState<Partial<Invitation>>({
+  // use an array to later allow sending multiple invitations
+  const [invitation, setInvitation] = useState<InvitationFieldInfoType>({
+    email: '',
     permission: PermissionLevel.Read,
   });
 
-  const isInvitationInvalid = ({ email }: { email?: string }) => {
+  const isInvitationInvalid = ({ email }: { email: string }): string => {
```

Comment:
Change the function name since it does not return a boolean anymore? Also, instead of returning an empty string if the conditions are not met, what about typing the function as returning `string | undefined` or `string | null`?